### PR TITLE
Problem Suggestion: Add "Friendly Spiders" to Silver - Additional Practice for USACO Silver

### PR DESCRIPTION
**Pull request by maggieliu05**
User `XRtEVBniu0RHuspDMz0rR5ux2l02` suggested adding the problem [Friendly Spiders](https://codeforces.com/contest/1775/problem/D) to the `problems` table of the module [Silver - Additional Practice for USACO Silver](https://usaco.guide/silver/silver-conclusion#problemlist-problems).

**Automatically Generated JSON:**
```json
{
  "uniqueId": "cf-1775D",
  "name": "Friendly Spiders",
  "url": "https://codeforces.com/contest/1775/problem/D",
  "source": "CF",
  "difficulty": "Normal",
  "isStarred": false,
  "tags": [
    "Number theory",
    "Graphs",
    "BFS",
    "shortest path"
  ],
  "solutionMetadata": {
    "kind": "autogen-label-from-site",
    "site": "CF"
  }
}
```
**Additional Notes**: None

*This PR was automatically generated from a user-submitted problem suggestion on the USACO guide.*

### CODE REVIEW

**Review by envyaims — APPROVED** [review]

File: content/3_Silver/Conclusion.problems.json [hunk 1]
```diff
@@ -202,6 +202,19 @@
...
       }
     },
+    {
+      "uniqueId": "cf-1775D",
+      "name": "Friendly Spiders",
+      "url": "https://codeforces.com/contest/1775/problem/D",
+      "source": "CF",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Number theory", "Graphs", "BFS", "shortest path"],
```
Comment (envyaims) [inline content/3_Silver/Conclusion.problems.json:212]:
```suggestion
      "tags": ["Number theory",  "BFS"],
```

**Review by envyaims — APPROVED** [review]

File: content/3_Silver/Conclusion.problems.json (hunk 1 above)
Comment (SansPapyrus683) [reply content/3_Silver/Conclusion.problems.json:212]: "NT" is more common as a tag

**Comment by ryanchou-dev** [comment]
this looks good to me, any thoughts @SansPapyrus683 ?

**Comment by SansPapyrus683** [comment]
image
joever

**Pull request merged**